iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -128,9 +128,7 @@ public enum ErrorCode
         JobQuotaExceeded = 403003,
 
         /// <summary>
-        /// The underlying cause is that the number of messages enqueued for the device exceeds the queue limit (50).
-        /// The most likely reason that you're running into this limit is because you're using HTTPS to receive the message,
-        /// which leads to continuous polling using ReceiveAsync, resulting in IoT hub throttling the request.
```

Comment:
The statement about HTTP polling for receive operations causing requests to be throttled is irrelevant to device queue depth exceeded error code. Let me know if you had something else in mind while adding this.

Reply:
I don't follow what this error was getting at either. I'm in favor of deleting it as well